machine/accton/accton_as7412_32x/installer.conf

```diff
@@ -0,0 +1,31 @@
+# Accton as7412_32x ONIE installer configuration
+
+description="Accton, as7412_32x"
+
+mass_dev="storage-not-found"
+discover_mass_storage()
+{
+    mass_bus="$1"
+    for i in a b c d e ; do
+        if $(ls -l /sys/block/sd$i/device 2>/dev/null | grep -q "$mass_bus") ; then
+            mass_dev="/dev/sd$i"
+            return 0
+        fi
+    done
+    return 1
+}
+
+# Default ONIE block device
+install_device_platform()
+{
+    discover_mass_storage "1-1.4:1.0" ||
+    discover_mass_storage "target4:0:0"
```

Comment:
The order here is opposite that in the patches for AS7712_16X and AS5822_54X.  Is that correct?  I figured they would all 3 be the same.